deepmd/dpmodel/descriptor/se_e2_a.py

```diff
@@ -189,15 +189,16 @@ def __init__(
             ndim=(1 if self.type_one_side else 2),
             network_type="embedding_network",
         )
-        for embedding_idx in itertools.product(
-            range(self.ntypes), repeat=self.embeddings.ndim
+        for ii, embedding_idx in enumerate(
+            itertools.product(range(self.ntypes), repeat=self.embeddings.ndim)
         ):
             self.embeddings[embedding_idx] = EmbeddingNet(
                 in_dim,
                 self.neuron,
                 self.activation_function,
                 self.resnet_dt,
                 self.precision,
+                seed=seed + len(self.neuron) * ii if seed is not None else None,
```

Comment:
shall we define a class method telling the number of random seeds used by one embedding net?

Reply:
Now it accepts a `list[int]`, so this is not needed anymore.